### PR TITLE
New:  Add `verticalCellBorder` as a prop for PagedGrid

### PR DESCRIPTION
**Pull request by vinteo**

### CODE REVIEW

File: test/components/adslotUi/PagedGridComponentTest.js [hunk 1]
```diff
@@ -37,10 +37,12 @@ describe('PagedGridComponent', () => {
       columns: [{ key: 'name' }],
       items: [{ name: 'item1' }, { name: 'item2' }, { name: 'item3' }],
       perPage: 1,
+      verticalCellBorder: true,
     };
     const component = shallow(<PagedGridComponent {...props} />);
     const gridRowElements = component.find(GridRow);
     expect(gridRowElements).to.have.length(2);
+    gridRowElements.forEach((gridRowElement) => expect(gridRowElement.prop('verticalCellBorder')).to.equal(true));
```
Comment (OndrejCholeva) [inline test/components/adslotUi/PagedGridComponentTest.js]: I'd be happy if you check the the default `off` case as well, LGTM otherwise.

**Comment by omgaz** [comment]
# LGTM 🍥

**Comment by OndrejCholeva** [comment]
LGTM :vertical_traffic_light:

**Comment by vinteo** [comment]
@omgaz @OndrejCholeva merge?

**Comment by OndrejCholeva** [comment]
merging :rocket:

**Pull request merged**